src/CommandingOfficers/AW2/AW2Commander.java

```diff
@@ -0,0 +1,59 @@
+package CommandingOfficers.AW2;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.BuggedEnergyCostCommander;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameScenario;
+import Units.Unit;
+import Units.UnitDelta;
+
+public abstract class AW2Commander extends BuggedEnergyCostCommander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage AW2_MECHANICS_BLURB = new InfoPage(
+      "COP charge gained from combat is funds damage taken and half of funds damage dealt.\n"
+    + TRILOGY_MECHANICS_BLURB.info
+    );
+
+  public AW2Commander(CommanderInfo info, GameScenario.GameRules rules)
+  {
+    super(info, rules);
+  }
+
+  @Override
+  public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCounter)
```

Comment:
You moved this down from `TrilogyCommander` - doesn't that mean that `AW3Commander` now inherits the Default `Commander`'s method for this (which counts both HP and cost for generating power)?

Reply:
Never mind, found it. It just didn't show up in the other diff because it was already present.